packages/transaction-controller/src/utils/batch.test.ts

```diff
@@ -1450,6 +1504,8 @@ describe('Batch Utils', () => {
 
         const result = await resultPromise;
         expect(result?.batchId).toMatch(/^0x[0-9a-f]{32}$/u);
+        expect(simulateGasBatchMock).toHaveBeenCalledTimes(1);
+        expect(getGasFeesMock).toHaveBeenCalledTimes(1);
```

Comment:
Should this warrant dedicated separate unit tests to verify that gas limit and gas fees are actually set on the batch metadata in the state?